src/main/java/com/springbootfinancetracker/dto/FinancialGoalDto.java

```diff
@@ -6,7 +6,7 @@
 @Data
 public class FinancialGoalDto {
     private Integer goalID;
-    private Integer targetamount;
+    private Integer targetAmount;
```

Comment:
Renamed targetamount variable to match the camel case of other variable names and follow best practice.